language/Portuguese/strings.po

```diff
@@ -20,157 +20,166 @@ msgstr ""
 #: src/frontend/mame/ui/slotopt.cpp:192
 #, c-format
 msgid "%s [internal]"
-msgstr ""
+msgstr "%s [interna]"
 
 #: src/frontend/mame/ui/slotopt.cpp:203 src/frontend/mame/ui/inputmap.cpp:625
 #: src/frontend/mame/ui/miscmenu.cpp:96
 msgid "Reset"
-msgstr ""
+msgstr "Reiniciar"
 
 #: src/frontend/mame/ui/slotopt.cpp:220 src/frontend/mame/ui/filesel.cpp:257
 #: src/frontend/mame/ui/swlist.cpp:83
 msgid "[empty slot]"
-msgstr ""
+msgstr "[slot vazio]"
 
 #: src/frontend/mame/ui/filesel.cpp:261
 msgid "[create]"
-msgstr ""
+msgstr "[criar]"
 
 #: src/frontend/mame/ui/filesel.cpp:265 src/frontend/mame/ui/swlist.cpp:94
 msgid "[software list]"
-msgstr ""
+msgstr "[lista de software]"
 
 #: src/frontend/mame/ui/filesel.cpp:525
 msgid "Select access mode"
-msgstr ""
+msgstr "Seleccionar modo de acesso"
 
 #: src/frontend/mame/ui/filesel.cpp:526
 msgid "Read-only"
-msgstr ""
+msgstr "Apenas-leitura"
 
 #: src/frontend/mame/ui/filesel.cpp:528
 msgid "Read-write"
-msgstr ""
+msgstr "Leitura-e-escrita"
 
 #: src/frontend/mame/ui/filesel.cpp:529
 msgid "Read this image, write to another image"
-msgstr ""
+msgstr "Ler esta imagem, escrever para outra imagem"
 
 #: src/frontend/mame/ui/filesel.cpp:530
 msgid "Read this image, write to diff"
-msgstr ""
+msgstr "Ler esta imagem, escrever para diff"
 
 #: src/frontend/mame/ui/info.cpp:30
 msgid "protection"
-msgstr ""
+msgstr "protecção"
 
 #: src/frontend/mame/ui/info.cpp:31
 msgid "color palette"
-msgstr ""
+msgstr "paleta de cores"
 
 #: src/frontend/mame/ui/info.cpp:32
 msgid "graphics"
-msgstr ""
+msgstr "gráficos"
 
 #: src/frontend/mame/ui/info.cpp:33
 msgid "sound"
-msgstr ""
+msgstr "som"
 
 #: src/frontend/mame/ui/info.cpp:34
 msgid "controls"
-msgstr ""
+msgstr "controlos"
 
 #: src/frontend/mame/ui/info.cpp:35
 msgid "keyboard"
-msgstr ""
+msgstr "teclado"
 
 #: src/frontend/mame/ui/info.cpp:36
 msgid "mouse"
-msgstr ""
+msgstr "rato"
 
 #: src/frontend/mame/ui/info.cpp:37
 msgid "microphone"
-msgstr ""
+msgstr "microfone"
 
 #: src/frontend/mame/ui/info.cpp:38
 msgid "camera"
-msgstr ""
+msgstr "câmara"
 
 #: src/frontend/mame/ui/info.cpp:39
 msgid "disk"
-msgstr ""
+msgstr "disco"
 
 #: src/frontend/mame/ui/info.cpp:40
 msgid "printer"
-msgstr ""
+msgstr "impressora"
 
 #: src/frontend/mame/ui/info.cpp:41
 msgid "LAN"
-msgstr ""
+msgstr "rede local"
 
 #: src/frontend/mame/ui/info.cpp:42
 msgid "WAN"
-msgstr ""
+msgstr "WAN"
 
 #: src/frontend/mame/ui/info.cpp:43
 msgid "timing"
-msgstr ""
+msgstr "temporização"
 
 #: src/frontend/mame/ui/info.cpp:180
 msgid ""
 "One or more ROMs/CHDs for this machine are incorrect. The machine may not "
 "run correctly.\n"
 msgstr ""
+"Uma ou mais ROMs/CHDs para esta máquina estão incorretas. A máquina poderá "
+"não funcionar corretamente"
```

Comment:
This string is missing the final `\n` which will mess up formatting - could you please add it?

Reply:
Sure.